ELEMENTS/Bromine.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: page
+title: ELEMENT_NAME
+---
+
+Symbol  : Br
+Atomic Number : 35
+Description : It is a volatile red-brown liquid at room temperature that evaporates readily to form a similarly coloured vapour.
```

Comment:
The rubric requires three sentences about bromine.

Reply:
Thank you for pointing this out!